.gitmodules

```diff
@@ -0,0 +1,6 @@
+[submodule "dist/plugins/PluginBase"]
```

Comment:
Please add a remark to the developer docs, that `git submodules` something is required to work with Roddy.